pkg/proxy/iptables/proxier.go

```diff
@@ -466,16 +487,33 @@ func NewProxier(ipt utiliptables.Interface,
 
 	if len(clusterCIDR) == 0 {
 		glog.Warningf("clusterCIDR not specified, unable to distinguish between internal and external traffic")
+	} else {
+		// Filter out the incorrect IP version case.
```

Comment:
Rather than filtering - this should probably be a real error.  What do you think?

Reply:
Yes, this should be a real error. Done.